benchmark_scenarios/eurac_pv_farm_detection.json

```diff
@@ -1,27 +1,36 @@
 [
-    {
-      "id": "eurac_pv_farm_detection",
-      "type": "openeo",
-      "description": "ML photovoltaic farm detection, developed by EURAC",
-      "backend": "openeofed.dataspace.copernicus.eu",
-      "process_graph": {
-        "maxndvi1": {
-          "process_id": "eurac_pv_farm_detection",
-          "namespace": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/4003046e3b79ec3ab8dace888a231655db389d66/openeo_udp/eurac_pv_farm_detection/eurac_pv_farm_detection.json",
-          "arguments": {
-            "bbox": {
-                "east": 16.414,
-                "north": 48.008,
-                "south": 47.962,
-                "west": 16.342
-            },
-            "temporal_extent": [
-              "2023-05-01",
-              "2023-09-30"
-            ]
+  {
+    "id": "eurac_pv_farm_detection",
+    "type": "openeo",
+    "description": "ML photovoltaic farm detection, developed by EURAC",
+    "backend": "openeofed.dataspace.copernicus.eu",
+    "process_graph": {
+      "maxndvi1": {
+        "process_id": "eurac_pv_farm_detection",
+        "namespace": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/4003046e3b79ec3ab8dace888a231655db389d66/openeo_udp/eurac_pv_farm_detection/eurac_pv_farm_detection.json",
+        "arguments": {
+          "bbox": {
+              "east": 16.414,
+              "north": 48.008,
+              "south": 47.962,
+              "west": 16.342
           },
-          "result": true
-        }
+          "temporal_extent": [
+            "2023-05-01",
+            "2023-09-30"
+          ]
+        },
+        "result": true
       }
+    },
+    "job_options": {
+      "driver-memory": "1g",
+      "executor-memory": "1g",
+      "python-memory": "3g"}
+    ,
+    "reference_data": {
+    },
+    "reference_options": {
```

Comment:
If you leave "reference_options" empty, I would just omit it to keep things to the point